xunit compilation errors on test platform 1.6.0

## Describe the bug

Build error:

```
xunit.v3.core\1.1.0\buildTransitive\xunit.v3.core.targets(30,70): error MSB4057: The target "_IncludeGenerateAutoRegisteredExtensionsIntoCompilation" does not exist in the project.
```

After adding an empty target with the above name we get:

```
xunit.v3.core\1.1.0\_content\EntryPoint-mtp.cs(11,118): error CS0400: The type or namespace name 'SelfRegisteredExtensions' could not be found in the global namespace (are you missing an assembly reference?)
```

## Steps To Reproduce

Create a project and reference

```xml
    <PackageReference Include="Microsoft.Testing.Platform" Version="1.6.0" />
    <PackageReference Include="Microsoft.Testing.Platform.MSBuild" Version="1.6.0" />
    <PackageReference Include="Microsoft.Testing.Extensions.VSTestBridge" Version="1.6.0" />
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.13.0" />
    <PackageReference Include="xunit.runner.visualstudio" Version="3.0.2" />
    <PackageReference Include="xunit.v3" Version="1.1.0" />
```

## Expected behavior
Build successful

## Actual behavior
Above error

--- a/src/Platform/Microsoft.Testing.Platform.MSBuild/buildMultiTargeting/Microsoft.Testing.Platform.MSBuild.targets
+++ b/src/Platform/Microsoft.Testing.Platform.MSBuild/buildMultiTargeting/Microsoft.Testing.Platform.MSBuild.targets
@@ -151,10 +151,10 @@
       <FileWrites Include="$(_TestingPlatformEntryPointSourcePath)" />
     </ItemGroup>
   </Target>
-  <!-- Note: _IncludeGenerateTestingPlatformEntryPointIntoCompilation is set to depend on _IncludeGenerateSelfRegisteredExtensionsIntoCompilation -->
+  <!-- Note: _IncludeGenerateTestingPlatformEntryPointIntoCompilation is set to depend on _IncludeGenerateAutoRegisteredExtensionsIntoCompilation -->
   <!-- This is because file orders matter in F# -->
   <Target Name="_IncludeGenerateTestingPlatformEntryPointIntoCompilation" BeforeTargets="BeforeCompile;XamlPreCompile"
-          DependsOnTargets="_CalculateGenerateTestingPlatformEntryPoint;_GenerateTestingPlatformEntryPointFileInputCache;_GenerateTestingPlatformEntryPoint;_IncludeGenerateSelfRegisteredExtensionsIntoCompilation">
+          DependsOnTargets="_CalculateGenerateTestingPlatformEntryPoint;_GenerateTestingPlatformEntryPointFileInputCache;_GenerateTestingPlatformEntryPoint;_IncludeGenerateAutoRegisteredExtensionsIntoCompilation">
 
     <ItemGroup Condition=" '$(GenerateTestingPlatformEntryPoint)' == 'True' ">
       <Compile Include="$(_TestingPlatformEntryPointSourcePath)" />
@@ -242,7 +242,7 @@
 
   <!-- We always include the entry point also if the task _GenerateSelfRegisteredExtensions is skipped for caching reason -->
   <!-- !!! DO NOT CHANGE THE NAME OF THIS TARGET IS USED BY ADAPTERS https://github.com/microsoft/testfx/issues/3478#issuecomment-2313889212 !!! -->
-  <Target Name="_IncludeGenerateSelfRegisteredExtensionsIntoCompilation" BeforeTargets="BeforeCompile;XamlPreCompile"
+  <Target Name="_IncludeGenerateAutoRegisteredExtensionsIntoCompilation" BeforeTargets="BeforeCompile;XamlPreCompile"
           DependsOnTargets="_CalculateGenerateSelfRegisteredExtensions;_GenerateSelfRegisteredExtensionsFileInputCache;_GenerateSelfRegisteredExtensions">
 
     <ItemGroup Condition=" '$(GenerateSelfRegisteredExtensions)' == 'True' ">

